src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -384,6 +384,37 @@
               --connector-name aci-connector --chart-url <CustomURL> --image-tag <VirtualKubeletImageTag>
 """
 
+helps['aks use-dev-connect'] = """
```

Comment:
Update for new product name (when it's confirmed)? e.g. use-dev-spaces

Reply:
I will wait till it really get finalized :)